Designer/Program.cs

```diff
@@ -15,19 +15,45 @@ static void Close ()
 			MessageBox.ErrorQuery (50, 7, "Error", "There is nothing to close", "Ok");
 		}
 
+		static void Copy ()
+		{
+			TextField textField = menu.LastFocused as TextField;
+			if (textField != null && textField.SelectedLength != 0) {
+				textField.Copy ();
+			}
```

Comment:
I think this is fine for now, but generally, I think that we should have a global "Copy" operation that copies whatever the currently focused view wants to provide.

So we might need to introduce a "IInputView" interface or something like that, following the spirit of UIKit.

Reply:
I agree. I had already thought about it after having implemented it in TextField. Work for later.